src/AWS.Deploy.Orchestration/Utilities/ICommandLineWrapper.cs

```diff
@@ -41,6 +45,7 @@ public interface ICommandLineWrapper
             bool streamOutputToInteractiveService = true,
             Action<TryRunResult> onComplete = null,
             bool redirectIO = true,
+            string awsExecutionEnv = "",
```

Comment:
I'm thinking `awsExecutionEnv` makes more sense as a constructor parameter to `CommandLineWrapper` (there's a separate thread for that discussion).

However, _if_ you do keep `awsExecutionEnv` as method parameter, think you should also update `TryRunWithResults` for consistency.